<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="checkBox_autostart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>311</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Автоматичеки запускать УТМы при запуске программы</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autoexit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>401</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Автоматически закрывать приложение после успешного запуска УТМ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_ok">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_exit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_tray">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>371</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Запускать свернутым в трей</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
